resources/example-policies/cleanup-policies.yml

```diff
@@ -0,0 +1,29 @@
+policies:
+  - name: terminate-c7n-101-tagged-ec2s
+    resource: aws.ec2
+    filters:
+      - tag:c7n-101: present
+    actions:
+      - terminate
+      - type: tag
```

Comment:
Since we're deleting/terminating resources immediately (rather than delayed via `mark-for-op`), I don't think we'll have a chance to see these mark tags. We can probably scrap the tag actions.

Reply:
Per pair programming session: In order to prevent display of terminated resources when filtering for tag, will also filter for active resources.